implementations/rust/ockam/ockam_api/src/kafka/decoder.rs

```diff
@@ -0,0 +1,62 @@
+use crate::kafka::kafka_portal_worker::MAX_KAFKA_MESSAGE_SIZE;
+use bytes::BytesMut;
+use futures::Stream;
+use ockam::compat::tokio;
+use ockam_transport_tcp::MAX_PAYLOAD_SIZE;
+use std::io::{Error, ErrorKind};
```

Comment:
Can we safely replace this with no_std counterparts?

Reply:
fixed everything